### PR TITLE
tuxi: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]
Wow, such a nice little tool 👍🏻

File: pages/linux/tuxi.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# tuxi
+
+> A CLI tool that scrapes Google search results and SERPs that provides instant and concise answers.
+> More information: <https://github.com/Bugswriter/tuxi>.
+
+- Use Google to search for a string:
+
+`tuxi {{search_string}}`
+
+- Show [r]aw search results (no pretty output, no colors):
```
Comment (navarroaxel) [inline pages/linux/tuxi.md:10]: Perhaps is this better:
```suggestion
- Display the search results in [r]aw format (no pretty output, no colors):
```

File: pages/linux/tuxi.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`tuxi -r {{search_string}}`
+
+- Show only search results (silences "Did you mean?", greetings and usage):
+
+`tuxi -q {{search_string}}`
+
+- Display help and usage of `tuxi`:
```
Comment (navarroaxel) [inline pages/linux/tuxi.md:18]: I think `tuxi` is redundant in the example description.
```suggestion
- Display help:
```

File: pages/linux/tuxi.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Use Google to search for a string:
+
+`tuxi {{search_string}}`
+
+- Show [r]aw search results (no pretty output, no colors):
+
+`tuxi -r {{search_string}}`
+
+- Show only search results (silences "Did you mean?", greetings and usage):
```
Comment (navarroaxel) [inline pages/linux/tuxi.md:14]:
```suggestion
- Display only search results (silences "Did you mean?", greetings and usage):
```

File: pages/linux/tuxi.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# tuxi
+
+> A CLI tool that scrapes Google search results and SERPs that provides instant and concise answers.
+> More information: <https://github.com/Bugswriter/tuxi>.
+
+- Use Google to search for a string:
+
+`tuxi {{search_string}}`
```
Comment (navarroaxel) [inline pages/linux/tuxi.md:8]:
```suggestion
`tuxi {{search_terms}}`
```

File: pages/linux/tuxi.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# tuxi
+
+> A CLI tool that scrapes Google search results and SERPs that provides instant and concise answers.
+> More information: <https://github.com/Bugswriter/tuxi>.
+
+- Use Google to search for a string:
```
Comment (navarroaxel) [inline pages/linux/tuxi.md:6]: What if we start with a verb?
```suggestion
- Make a search using Google:
```
What do you think @bl-ue ?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**